app/webpacker/packs/school_search.js

```diff
@@ -0,0 +1,35 @@
+import accessibleAutocomplete from "accessible-autocomplete";
+
+const axios = require("axios");
+
+const element = document.querySelector("#school-search-form-autocomplete");
+const id = "school-search-form-input";
+
+function inputValueTemplate(result) {
+  return result && result.name;
+}
+
+function suggestionTemplate(result) {
+  return result && `<strong>${result.name}</strong> (${result.full_address_formatted})`;
+}
```

Comment:
Ah, crap, we have to compute this in js because autocomplete constructs its own html using js? argh. Oh well.

Reply:
Yep - took some banging on the monitor and quite a few swear words to figure it right